src/components/BeforeGame.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+import Input from './Input';
+import Button from './Button';
+
+import boom from '../assets/img/boom.png';
```

Comment:
to be consistent, it could be named `boomSrc`, as this variable keep source to boom, not boom itself